src/util.ts

```diff
@@ -286,3 +286,18 @@ export function replaceExt(
   const extToReplace = new RegExp(`\\${replaceExpandedExt ? expandedExt : baseExt}$`, 'i');
   return fileName.replace(extToReplace, newExtension);
 }
+
+/**
+ * This only removes `.js` from a web modules folder, which is the only known
+ * conflict. This was created to fix the “tippy.js problem”, where an npm
+ * package with an extension would fail to install because it would try and
+ * create both a `tippy.js` directory as well as a `tippy.js` file.
+ *
+ * This should only be used for web_modules, not source code (users should be
+ * able to name folders whatever they want)
+ */
+export function webModulesSubImportPath(filepath: string): string {
```

Comment:
Took me a second to read the comment and understand what this was doing. is there a clearer name that we could use? `gePackageImportPath`? `gePackageResourceImportPath`?

Reply:
Yeah, agreed it wasn’t clear at all. I was thinking about it again, and it really doesn’t have to do with importing or resources; it honestly only has to do with npm packages that end in `.js` and nothing else. What do you think about the change to `sanitizePackageName()`?

Reply:
Sorry—merged, but would happily take input on the name as a follow-up PR (I wanted to do some testing with the core of this PR merged)